core/dispatcher/src/main/scala/whisk/core/dispatcher/Dispatcher.scala

```diff
@@ -191,8 +195,6 @@ object Dispatcher extends Logging {
     def requiredProperties =
         Map(servicePort -> 8080.toString()) ++ kafkaHost
 
-    val executionContext = ExecutionContextFactory.makeCachedThreadPoolExecutionContext()
```

Comment:
Now constructing the `ActorSystem` with the aforementioned execution context in `InvokerService.main`.